octobot_trading/personal_data/portfolios/portfolio_value_holder.py

```diff
@@ -261,21 +291,74 @@ def _try_get_value_of_currency(self, currency, quantity, raise_error):
                     raise missing_data_exception
         return constants.ZERO
 
-    def convert_currency_value_using_last_prices(self, quantity, current_currency, target_currency):
+    def convert_currency_value_using_last_prices(
+        self, quantity, current_currency, target_currency, settlement_asset=None
+    ):
         try:
-            symbol = symbol_util.merge_currencies(current_currency, target_currency)
+            symbol = symbol_util.merge_currencies(
+                current_currency, target_currency, settlement_asset=settlement_asset
+            )
             if self._has_price_data(symbol):
                 return quantity * self._get_last_price_data(symbol)
         except KeyError:
             pass
         try:
-            reversed_symbol = symbol_util.merge_currencies(target_currency, current_currency)
+            reversed_symbol = symbol_util.merge_currencies(
+                target_currency, current_currency, settlement_asset=settlement_asset
+            )
             return quantity / self._get_last_price_data(reversed_symbol)
-        except decimal.DivisionByZero:
-            pass
-        except KeyError:
+        except (KeyError, decimal.DivisionByZero):
             pass
-        raise errors.MissingPriceDataError(f"no price data to evaluate {current_currency} price in {target_currency}")
+        raise errors.MissingPriceDataError(
+            f"no price data to evaluate {current_currency} price in {target_currency}"
```

Comment:
Will this error be displayed to the user?

Reply:
In rare cases in logs it can